src/Root.cpp

```diff
@@ -720,6 +756,27 @@ void cRoot::BroadcastPlayerListsAddPlayer(const cPlayer & a_Player, const cClien
 }
 
 
+
+
+
+size_t cRoot::GetMaxRAMChunks()
```

Comment:
xoft's insistence of 5 lines make the files too big and hard to scroll through. A getter is unlikely to affect your recompiles since they don't ever change.